### ISSUE TITLE
Docs for the front-end work

### ISSUE THREAD

**Issue by mariamrf** [blaringsilence/pegasus#31]
Docs currently available only cover the backend functions, but not the front-end architecture, which is an integral part of this project, so the currently available docs need to be edited to include the js logic. Everything should be explained in the code comments, though.

**Comment by shinning91** [comment]
So just adding comments in js files?

**Comment by mariamrf** [comment]
Not really; more like the documentation for the backend [here](http://Pegasus.readthedocs.io)
